File: src/AHF-API/DataLayer/Model/InventoryModel/Inventory.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DataLayer.Model.InventoryModel
{
    public class Inventory
    {
        public Inventory()
        {
            Items = new List<InventoryItem>();
            Ingredients = new List<InventoryItem>();
        }

        public int Id { get; set; }
        public List<InventoryItem> Items { get; set; }
        public List<InventoryItem> Ingredients { get; set; }
    }
}

File: src/AHF-App/app/src/main/java/com/ahf/antwerphasfallen/Model/Inventory.java
package com.ahf.antwerphasfallen.Model;

import java.util.List;

/**
 * Created by Jorren on 16/11/2018.
 */

public class Inventory {
    private int id;
    private List<InventoryItem> ingredients;
    private List<InventoryItem> items;

    public Inventory() {
    }

    public int getId() {
        return id;
    }

    public List<InventoryItem> getIngredients() {
        return ingredients;
    }

    public List<InventoryItem> getItems() {
        return items;
    }
}
